apps/client/src/common/components/multi-part-progress-bar/MultiPartProgressBar.tsx

```diff
@@ -0,0 +1,53 @@
+import { clamp } from '../../utils/math';
+
+import './MultiPartProgressBar.scss';
+
+interface MultiPartProgressBar {
+  now?: number;
+  complete?: number;
+  normalColor?: string;
+  warning?: number;
+  warningColor?: string;
+  danger?: number;
+  dangerColor?: string;
+  hidden?: boolean;
+  className?: string;
+  height?: string;
+}
+
+export default function MultiPartProgressBar(props: MultiPartProgressBar) {
+  const {
+    now = 0,
+    complete = 100,
+    normalColor,
+    warning = 85,
```

Comment:
i thought since you defined in your progress bar the defaults 0 and 100 (as percentages since they are handled in code as that) i add the defaults for warning and danger also as percentages

Reply:
since they aren't optional anymore i've removed the defaults